docs/api.md

```diff
@@ -52,6 +52,10 @@ PettingZoo models games as *Agent Environment Cycle* (AEC) games, and thus can s
 
 `action_spaces`: A dict of the action spaces of every agent, keyed by name. This cannot be changed through play or reseting.
 
+`state_space`: The space of a global observation of the environment. Currently, only `butterfly` environments have this attribute implemented. If `state_space` is called in an unsupported environment the following `AttributeError` will raise:
+
+`Environment {} has no state_space attribute implemented yet.`
```

Comment:
this should be in the same line as the rest

Reply:
Actually, including the specific error message isn't really needed. in the documentation at all, just mention that not all environments will support it (in many cases it doesn't even make sense to support this, like with Atari). Relatedly, "yet" should be removed from the error message.